<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCColonos</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(127, 4, 24);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffbb00;&quot;&gt;SELECCIONA TU MONOPOLIO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_arcilla">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>ARCILLA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_madera">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>MADERA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_trigo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>TRIGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/Desarrollo/monopolio_1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
